tutorials/analysis/vonBertalanffy/tutorial_vonBertalanffy.m

```diff
@@ -42,17 +46,26 @@
 modelFileName = [modelName '.mat']
 modelDirectory = getDistributedModelFolder(modelFileName); %Look up the folder for the distributed Models.
 modelFileName= [modelDirectory filesep modelFileName]; % Get the full path. Necessary to be sure, that the right model is loaded
-model = readCbModel(modelFileName);
+
 switch modelName
     case 'iAF1260'
+        model = readCbModel(modelFileName);
         if model.S(952, 350)==0
             model.S(952, 350)=1; % One reaction needing mass balancing in iAF1260
         end
         model.metCharges(strcmp('asntrna[c]', model.mets))=0; % One reaction needing charge balancing
 
-    case 'Recon3.0model'
-        model.metFormulas{strcmp(model.mets,'h[i]')}='H';
-        model.metFormulas(cellfun('isempty',model.metFormulas)) = {'R'};
+    case 'Recon3D_Dec2017'
+      model = readCbModel(modelFileName);
+      model.csense(1:size(model.S,1),1)='E';
```

Comment:
Any reason for `model.csense(1:size(model.S,1),1)='E'`?

Reply:
As below, this was copied from the pdf with instructions.